lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/EagerFallback.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <ATen/core/ivalue.h>
+#include <ATen/core/stack.h>
+
+namespace torch_lazy_tensors {
+
+void eager_fallback(const c10::OperatorHandle& op, torch::jit::Stack* stack,
```

Comment:
Nit: `c10::Backend` is a legacy thing that I think we're trying to remove completely in the near-mid future. `c10::DeviceType` is probably better here

Reply:
Also, I remember you saying over chat the `eager_fallback()` can just fully replace the existing `at::native::cpu_fallback()`. Totally agreed, and feel free to make that change either as part of this PR or later during the merge

Reply:
Oof, I had a version with `c10::DeviceType` but replace it with `c10::Backend` because it lets me derive the dispatch key from it. I'll change it back.

Reply:
Done, restored the `c10::DeviceType` version.